.stylelintrc

```diff
@@ -1,29 +1,3 @@
 {
-  "extends": "stylelint-config-sass-guidelines",
-  "ignoreFiles": [
-    "demo/dist/main.css",
-    "nebula-components/tools/**/*.scss"
-  ],
-  "plugins": [
-    "stylelint-scss",
-    "stylelint-selector-bem-pattern"
-  ],
-  "rules": {
-    "selector-class-pattern": "",
-    "order/properties-alphabetical-order": null,
-    "selector-no-qualifying-type": [ true, {
-      "ignore": ["attribute"]
-    }],
-    "scss/selector-no-redundant-nesting-selector": true,
-    "scss/at-extend-no-missing-placeholder": true,
-    "scss/dollar-variable-no-missing-interpolation": true,
-    "scss/at-mixin-argumentless-call-parentheses": "never",
-    "plugin/selector-bem-pattern": {
-      "preset": "bem",
-      "componentName": "(([a-z0-9]+(?!-$)-?)+)",
-      "componentSelectors": {
-        "initial": "\\.{componentName}(((__|--)(([a-z0-9\\[\\]'=]+(?!-$)-?)+))+)?$"
-      }
-    }
-  }
+  "extends": "stylelint-config-nebula"
```

Comment:
Looks like I have to do some work on this stylelint-config to get our linting to pass...